README.md

```diff
@@ -13,7 +13,7 @@ data exploration to Elixir.
 
 Explorer high-level features are:
 
-- Simply typed series: `:binary`, `:boolean`, `:category`, `:date`, `:datetime`, `:duration`, `:float`, `:integer`, `:string`, and `:time`.
+- Simply typed series: `:binary`, `:boolean`, `:category`, `:date`, `:datetime`, `:duration`, `{:f, 32}`, `{:f, 64}`, `:integer`, `:string`, and `:time`.
```

Comment:
I wonder if we shouldn't do something like:
```suggestion
- Simply typed series: `:binary`, `:boolean`, `:category`, `:date`, `:datetime`, `:duration`, `:float` (`{:f, 32}` or `{:f, 64}`), `:integer`, `:string`, and `:time`.
```

WDYT?

Reply:
Agreed! I added more mentions of it in other places as well: https://github.com/elixir-explorer/explorer/pull/739/commits/3605cfcc7ca6877ea7568626d91b933e97148068